.pylintrc

```diff
@@ -4,6 +4,11 @@
 # there seems to be an off by one issue in how max-nested-blocks is used.
 max-nested-blocks = 2
 
+[FORMAT]
```

Comment:
What do you mean exactly? I just followed David's initial pull request to change default values.